Gemfile

```diff
@@ -33,6 +33,8 @@ group :development, :test do
 end
 
 group :development do
+  gem 'better_errors'
```

Comment:
Why are these being added? I was thinking Rails 4 has some built-in functionality which makes better_errors not so much better. I don't know what binding_of_caller is.